packages/desktop-client/src/components/common.tsx

```diff
@@ -510,9 +629,18 @@ export function Menu({ header, footer, items: allItems, onMenuSelect }) {
   );
 }
 
-Menu.line = Symbol('menu-line');
+const MenuLine: unique symbol = Symbol('menu-line');
+Menu.line = MenuLine;
 Menu.label = Symbol('menu-label');
```

Comment:
why the different handling here?

Reply:
Typescript being annoying... Apparently there is a different handling between type `symbol` and `unique symbol`, where [unique symbol](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#unique-symbol) allows `typeof Menu.line` (and subsequent equality checks). Without it could not find a way to have filtering work in the component 😞